src/js/providers/html5.js

```diff
@@ -419,9 +420,13 @@ define([
                 return;
             }
             _videotag.removeAttribute('src');
-            if (!_isIE) {
+            if (!_isMSIE) {
                 _videotag.load();
             }
+            // IE may continue to play a video after changing source and loading a new media file.  JW7-1705
+            if(utils.isIETrident()) {
```

Comment:
The isIETrident check is checking if we're Edge or Trident, since this issue exists in both i think we can just use utils.isIETrident.  I'll post the question and remove the story link.